graphql/schema.graphql

```diff
@@ -1,6 +1,25 @@
 type Query {
   userPatches(userId: String!): [Patch]!
   task(taskId: String!): Task
+  taskTests(
+    taskId: String!
+    sortCategory: TaskSortCategory!
```

Comment:
Shouldn't the params that aren't taskID be nullable, but have a default?

Reply:
The UI code should always provide values for all of these fields. Based on the use case in the most recent designs, if the UI code doesn't provide a value for all of these fields, then there may be something incorrect happening in the frontend which is why I made them required. I agree the query will be easier for external consumers if some of these fields were not required.

Reply:
Just pushed changes to add default params